nixos/modules/services/web-apps/glitchtip.nix

```diff
@@ -0,0 +1,283 @@
+  );
+
+  celeryArgs = builtins.concatStringsSep " " cfg.celery.extraArgs;
+
+  environment = lib.mapAttrs (
+    _: value:
+    if value == true then
+      "True"
+    else if value == false then
+      "False"
+    else
+      toString value
+  ) cfg.settings;
+in
+
+{
+  meta.maintainers = with lib.maintainers; [
+    defelo
+    felbinger
+  ];
+
+  options = {
+    services.glitchtip = {
+      enable = lib.mkEnableOption "Glitchtip";
+      package = lib.mkPackageOption pkgs "glitchtip" { };
+
+      user = lib.mkOption {
+        type = lib.types.str;
+        default = "glitchtip";
+        description = "User account under which glitchtip runs.";
+      };
+
+      group = lib.mkOption {
+        type = lib.types.str;
+        default = "glitchtip";
+        description = "Group under which glitchtip runs.";
+      };
+
+      listenAddress = lib.mkOption {
+        type = lib.types.str;
+        default = "localhost";
+        description = "The address to listen on.";
+      };
+
+      port = lib.mkOption {
+        type = lib.types.port;
+        default = 8000;
+        description = "The port to listen on.";
+      };
+
+      settings = lib.mkOption {
+        description = ''
+          Configuration of Glitchtip. See <https://glitchtip.com/documentation/install#configuration> for more information.
+        '';
+        default = { };
+        example = {
+          SECRET_KEY = "changeme";
+          GLITCHTIP_DOMAIN = "https://glitchtip.example.com";
+          DATABASE_URL = "postgres://postgres:postgres@postgres/postgres";
+        };
+
+        type = lib.types.submodule {
+          freeformType =
+            with lib.types;
+            attrsOf (oneOf [
+              str
+              int
+              bool
+            ]);
+
+          options = {
+            GLITCHTIP_DOMAIN = lib.mkOption {
+              type = lib.types.str;
+              description = "Domain under which GlitchTip is expected to be served on. Include scheme (http or https).";
+              example = "https://glitchtip.example.com";
+            };
+
+            ENABLE_USER_REGISTRATION = lib.mkOption {
+              type = lib.types.bool;
+              description = ''
+                When true, any user will be able to register. When false, user self-signup is disabled after the first user is registered. Subsequent users must be created by a superuser on the backend and organization invitations may only be sent to existing users.
+              '';
+              default = true;
+            };
+
+            ENABLE_ORGANIZATION_CREATION = lib.mkOption {
+              type = lib.types.bool;
+              description = ''
+                When false, only superusers will be able to create new organizations after the first. When true, any user can create a new organization.
+              '';
+              default = false;
+            };
+          };
+        };
+      };
+
+      extraPythonPackages = lib.mkOption {
```

Comment:
Why would I need this?

Reply:
We discussed about this too, the previous pr introduced it. We will remove it.